### ISSUE TITLE
Travis CI only checks python 3.7.5 on macOS

### ISSUE THREAD

**Issue by NickCrews** [shaypal5/cachier#23]
Hi! I notice that at in [.travis.yaml](https://github.com/shaypal5/cachier/blob/628f98e7c364241011ddcdc7c2bb6cc4e5a265fc/.travis.yml#L22) it is claimed that cachier is tested against Python 3.6.5 on macOS 10.13.

However, on a corresponding build on [Travis](https://travis-ci.org/github/shaypal5/cachier/jobs/677660460), if you look in the logs, you can see that it's actually using python 3.7.5. This is the same for another [build](https://travis-ci.org/github/shaypal5/cachier/jobs/677660461) that claims to use python 3.7.3, but also is using 3.7.5

I'm not sure where this changed.

**Comment by shaypal5** [comment]
Hmmm.
Let me check this!

**Comment by shaypal5** [comment]
Yup. This used to work - Travis even pulled my blog post on this to their own official blog - but I guess they keep rebuilding the images, and so now even old xcode images come with Python 3.7.5.

This is the image that gets pulled by the job, not something we can control. At the time, when I was figuring this out, this was the only way I found to easily test different Python versions on macOS. I think building it from source each time is too expensive an overhead, so we can bare to not test this on earlier Pythons specifically on macOS (they ARE tested on linux, after all).

I did use this chance to add testing for Python 3.8.0 on macOS, though!
